xml/libvirt_migrating_vms.xml

```diff
@@ -0,0 +1,623 @@
+          should be permanent (default) or temporary, using <guimenu>Temporary
+          move</guimenu>.
+        </para>
+        <para>
+          Additionally, there is the option <guimenu>Allow unsafe</guimenu>,
+          which allows migrating without disabling the cache of the &vmhost;.
+          This can speed up the migration but only works when the current
+          configuration allows for a consistent view of the &vmguest; storage
+          without using
+          <literal>cache=&quot;none&quot;</literal>/<literal>0_DIRECT</literal>.
+        </para>
+        <note>
+          <title>Bandwidth option</title>
+          <para>
+            In recent versions of &vmm;, the option of setting a bandwidth for
+            the migration has been removed. To set a specific bandwidth, use
+            <command>virsh</command> instead.
+          </para>
+        </note>
+      </step>
+      <step>
+        <para>
+          To perform the migration, click <guimenu>Migrate</guimenu>.
+        </para>
+        <para>
+          When the migration is complete, the <guimenu>Migrate</guimenu> window
+          closes and the &vmguest; is now listed on the new host in the &vmm;
+          window. The original &vmguest; is still available on the source host
+          in the shut-down state.
+        </para>
+      </step>
+    </procedure>
+  </sect1>
+  <sect1 xml:id="sec-libvirt-admin-migrate-virsh">
+    <title>Migrating with <command>virsh</command></title>
+
+    <para>
+      To migrate a &vmguest; with <command>virsh</command>
+      <option>migrate</option>, you need to have direct or remote shell access
+      to the &vmhost;, because the command needs to be run on the host. The
+      migration command looks like this:
+    </para>
+
+<screen>&prompt.user;virsh migrate [OPTIONS] <replaceable>VM_ID_or_NAME</replaceable> <replaceable>CONNECTION_URI</replaceable> [--migrateuri tcp://<replaceable>REMOTE_HOST:PORT</replaceable>]</screen>
+
+    <para>
+      The most important options are listed below. See <command>virsh help
+      migrate</command> for a full list.
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>--live</option></term>
+        <listitem>
+          <para>
+            Does a live migration. If not specified, the guest is paused during
+            the migration (<quote>non-live migration</quote>).
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--suspend</option></term>
+        <listitem>
+          <para>
+            Does a non-live migration and does not restart the &vmguest; on the
+            target host.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--persistent</option></term>
+        <listitem>
+          <para>
+            By default, a migrated &vmguest; is migrated temporarily, so its
```

Comment:
This isn't quite right either. --persistent will persist the VM on the destination host. Without the option, the VM is transient. I.e., it will not be seen by 'virsh list --all' when shutdown. To remove the VM on the source host, use the --undefinesource option.